Sources/Sentry/SentryHub.m

```diff
@@ -83,14 +83,15 @@ - (void)closeCachedSession {
     SentryFileManager *fileManager = [[self getClient] fileManager];
     SentrySession *session = [fileManager readCurrentSession];
     if (nil != session) {
+        [SentryLog logWithMessage:[NSString stringWithFormat:@"A cached session was found."] andLevel:kSentryLogLevelDebug];
```

Comment:
`l`: I think we don't need the `stringWithFormat` here. The same applies for the other logs in this method.
```suggestion
        [SentryLog logWithMessage:@"A cached session was found." andLevel:kSentryLogLevelDebug];
```